skeletons/README.md

```diff
@@ -0,0 +1,88 @@
+# Skeleton components
+
+## Skeleton design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:skeletons:<versions>")
+```
+
+### Rectangular
+
+
+```kotlin
+object VitaminSkeleton {
+    @Composable
+    fun Rectangular(
+        modifier: Modifier = Modifier,
+        colors: SkeletonColors = VitaminSkeletonColors.primary(),
+        cornerRadius: Dp = 0.dp
+    )
+}
+```
+
+This component draw a rectangle. You can add a modifier, but you can also customize the colors and the corner radius of the shape.
+
+```kotlin
+VitaminSkeleton.Rectangular()
+```
+
+Parameters | Descriptions
+-- | --
+`modifier: Modifier = Modifier` | The `Modifier` to be applied to the component
+`colors: SkeletonColors = VitaminSkeletonColors.primary()` | colors The colors of the background and the highlight effect
```

Comment:
remove `colors` keyword at the beginning

Reply:
same for all variants

Reply:
alright